### ISSUE TITLE
Scrub Unicode from sensor readings

### ISSUE THREAD

**Issue by aaaaalbert** [SensibilityTestbed/repy_v2#5]
[As seen during testing](https://github.com/SensibilityTestbed/repy_v2/commit/8f354acb5b87ee86feb99192e28c9bc652c03845#commitcomment-20730267), sensors such as the WiFi scanner may return Unicode strings. Unicode isn't allowed by the Repy sandbox, so that the namespace wrapping layer raises a very low-level error:
```
Exception (with class 'exception_hierarchy.InternalRepyError'):
Function 'get_wifi_connection_info' returned with unallowed return
type <type 'dict'> : _copy failed on {'ssid': u'".... \xe2\x9d\xa4 ...." (etc:)}
with message 'ascii' codec can't encode characters in position 13-15:
ordinal not in range(128)
```

We could use the existing [Unicode scrubber](https://github.com/SensibilityTestbed/sensorlib/blob/master/unicode_scrubber.r2py), or alternatively resort to a pure-Python solution that gets added between the namespace interface and the CPython sensor extension. Or do it in Java right away.

**Comment by aaaaalbert** [comment]
See 3217147717d6d4a4d62244ff3bc83be08398f4ee for a relatively complete implementation.

**Comment by aaaaalbert** [comment]
Fixed in e9303d60e16b06378aaaa2568783dd2e9367ba25. Clösing.
